Python/initconfig.c

```diff
@@ -280,39 +254,27 @@ static const char usage_envvars[] =
 "PYTHON_HISTORY  : the location of a .python_history file.\n"
 "\n"
 "These variables have equivalent command-line options (see --help for details):\n"
-"PYTHON_CPU_COUNT: Overrides the return value of os.process_cpu_count(),\n"
-"                  os.cpu_count(), and multiprocessing.cpu_count() if set to\n"
-"                  a positive integer. (-X cpu_count)\n"
+"PYTHON_CPU_COUNT: override the return value of os.cpu_count() (-X cpu_count)\n"
 "PYTHONDEBUG     : enable parser debug mode (-d)\n"
-"PYTHONDEVMODE   : enable the development mode (-X dev)\n"
+"PYTHONDEVMODE   : enable the Python development mode (-X dev)\n"
 "PYTHONDONTWRITEBYTECODE: don't write .pyc files (-B)\n"
 "PYTHONFAULTHANDLER: dump the Python traceback on fatal errors (-X faulthandler)\n"
-"PYTHON_FROZEN_MODULES: if this variable is set, it determines whether or not\n"
-"                  frozen modules should be used.  The default is \"on\" (or\n"
-"                  \"off\" if you are running a local build).\n"
-"                  (-X frozen_modules)\n"
+"PYTHON_FROZEN_MODULES: whether or not frozen modules should be used.\n"
```

Comment:
"or not" is redundant, remove full stop, make more direct/concise:

```suggestion
"PYTHON_FROZEN_MODULES: whether to use frozen modules\n"
```